pact/Pact/Core/IR/Eval/CoreBuiltin.hs

```diff
@@ -1229,17 +1279,25 @@ coreIntToStr info b cont handler _env = \case
 
 coreStrToInt :: (CEKEval step b i m, MonadEval b i m) => NativeFunction step b i m
 coreStrToInt info b cont handler _env = \case
-  [VString s] ->
-    checkLen info s *> doBase info cont handler 10 s
+  [VString s] -> do
+    chargeGasArgs info $ GStrOp $ StrOpLength $ T.length s
```

Comment:
I'm curious what we're charging for here? Shouldn't we be charging for parsing?

Reply:
ditto, updated